automatic/1password/update.ps1

```diff
@@ -15,36 +14,50 @@ function global:au_SearchReplace {
   }
 }
 
-function global:au_AfterUpdate {
+function global:au_BeforeUpdate {
   . "$PSScriptRoot/update_helper.ps1"
-  if ($Latest.PackageName -eq '1password4') {
     removeDependencies ".\*.nuspec"
```

Comment:
You can't do this in the `au_BeforeUpdate`, the changes will be overwritten when `au_SearchReplace` have ran.
It must be done in the `au_AfterUpdate` for the changes to be kept, please revert these changes.

Reply:
This was done to make it so `'1password' already has a dependency defined for 'dotnet4.6.1
Choco pack failed with exit code 1` doesn't happen locally outside of `Github`

This is also what happens in the nuspec file `<dependencies><dependency id="dotnet4.6.1" version="4.6.01055.20170308" /><dependency id="dotnet4.6.1" version="4.6.01055.20170308" /></dependencies>`

This fix prevents this error from occurring

Reply:
fair enough, but this style won't work either.
Here is what is going to be happening.
1. au_BeforeUpdate will remove the dependency, then save the nuspec file.
2. au_SearchReplace will run and save the file with the xml data that AU have cached, meaning it won't have detected the change that occurred during au_BeforeUpdate.

Reply:
removed due to inconsistent results